moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -2,163 +2,71 @@ export const ProjectActivityLogTableMaps = {
-        data_type: "date",
-      },
-      project_length: {
-        label: "length",
-        icon: "",
-        data_type: "integer",
-      },
-      fiscal_year: {
-        icon: "",
-        label: "fiscal year",
-        data_type: "text",
-      },
-      capitally_funded: {
-        icon: "",
-        label: "capitally funded",
-        data_type: "boolean",
-      },
-      project_priority: {
-        icon: "",
-        label: "priority",
-        data_type: "text",
-      },
-      date_added: {
-        label: "date added",
-        icon: "",
-        data_type: "timestamp with time zone",
-      },
-      added_by: {
-        icon: "",
-        label: "added by",
-        data_type: "integer",
-        lookup: {
-          table: "moped_users",
-          fieldLabel: "user_id",
-          fieldValues: ["first_name", "last_name"],
-        },
       },
       is_deleted: {
-        icon: "",
         label: "soft delete status",
-        data_type: "bool",
       },
       milestone_id: {
-        icon: "",
         label: "milestone ID",
-        data_type: "integer",
       },
       status_id: {
-        icon: "",
         label: "status ID",
-        data_type: "integer",
-      },
-      updated_at: {
-        icon: "",
-        label: "updated timestamp",
-        data_type: "timestamp with time zone",
       },
       // deprecated column, but keeping because historical activities depend on it
       contractor: {
-        icon: "",
         label: "contractor",
-        data_type: "text",
       },
       project_sponsor: {
-        icon: "",
-        label: "sponsor ID",
-        data_type: "integer",
+        label: "project sponsor",
+        lookup: "moped_entity",
       },
       project_website: {
-        icon: "",
         label: "project website",
-        data_type: "text",
       },
       knack_project_id: {
-        icon: "",
-        label: "knack internal ID",
-        data_type: "text",
+        label: "Knack internal ID",
       },
       // deprecated column, but keeping because historical activities depend on it
       purchase_order_number: {
-        icon: "",
         label: "purchase order number",
-        data_type: "text",
       },
       task_order: {
-        icon: "",
         label: "task order",
-        data_type: "jsonb",
       },
+      // deprecated column, but keeping because historical activities depend on it
       work_assignment_id: {
-        icon: "",
         label: "work assignment ID",
-        data_type: "text",
       },
       parent_project_id: {
-        icon: "",
         label: "parent project id",
-        data_type: "integer",
       },
+      interim_project_id: {
+        label: "Interim MPD (Access) ID",
+      },
+      project_lead_id: {
+        label: "project lead",
+        lookup: "moped_entity",
+      },
+      capitally_funded: {
+        label: "capitally funded"
+      },
+      is_retired: {
+        label: "is retired"
+      }
```

Comment:
I was testing other projects using the prod data and found a few more deprecated fields that have activity log entries

Reply:
interesting—so this is a lingering bug that exists in prod today? do you have any thoughts on how we could reliably avoid potential WSOD if other things fall through the cracks? shall i make a separate issue?

Reply:
It appears to be a regression I was going to introduce with this PR. Going to make sure I didnt lose any of the other fields when moving the table maps back to this file

Reply:
- https://github.com/cityofaustin/atd-data-tech/issues/11105